hazelcast/src/main/java/com/hazelcast/client/impl/protocol/codec/ClientAddClusterViewListenerCodec.java

```diff
@@ -37,45 +37,50 @@
 /**
  * TODO DOC
  */
-@Generated("5d1772412b87cda68e87d8ac98ba3bec")
+@Generated("31932e989ca413ed287b11a2e8f57174")
```

Comment:
Please merge your client protocol PR with `master` and run generator once again. This codec is not related with hot restart operations and it has changed because this repo and the client protocol one were out of sync. It was fixed by https://github.com/hazelcast/hazelcast-client-protocol/pull/277